Date:	Tue, 30 Jun 2009 22:39:28 -0500
From:	Larry Finger <Larry.Finger@lwfinger.net>
Subject: [PATCH V2] b43: Add support for 4318E

From: Clyde McPherson <ccmcphe@verizon.net>

Added support for the Broadcom 4318E chipset on PCMCIA/CF cards. The 
4318E can do 802.11A/B/G, only B and G mode are supported in b43.

Signed-off-by: Clyde McPherson <ccmcphe@verizon.net>
Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>

diff --git a/drivers/net/wireless/b43/pcmcia.c 
b/drivers/net/wireless/b43/pcmcia.c
index 3cfc303..6c3a749 100644
--- a/drivers/net/wireless/b43/pcmcia.c
+++ b/drivers/net/wireless/b43/pcmcia.c
@@ -35,6 +35,7 @@
 
 static /*const */ struct pcmcia_device_id b43_pcmcia_tbl[] = {
 	PCMCIA_DEVICE_MANF_CARD(0x2D0, 0x448),
+	PCMCIA_DEVICE_MANF_CARD(0x2D0, 0x476),
 	PCMCIA_DEVICE_NULL,
 };

